docs/dev-guides/personal-environments.md

```diff
@@ -469,6 +469,128 @@ gke-terra-integration-default-v2-95d71f95-gdfy   Ready    <none>   30h   v1.17.1
+```shell script
+$ ./render-k8s-config.sh zloery
+```
+
+<summary>testrunner-k8s-serviceaccount.yml.template</summary>
+
+```text
+# Do not modify this template file.
+
+# This template file is used for setting a Test Runner K8S Service Account
+# for running resiliency tests in a namespace.
+#
+# This template file is to be used in conjunction with the other template files
+#
+#   testrunner-k8s-role.yml.template
+#   testrunner-k8s-rolebinding.yml.template
+#
+# within an automation pipeline and is not meant to be run separately or manually.
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/component: APP
+  name: testrunner-k8s-sa
+  namespace: terra-NAMESPACE
+```
+
+<summary>testrunner-k8s-role.yml.template</summary>
+
+```text
+# Do not modify this template file.
+
+# This template file is used for setting a Test Runner privileged RBAC role
+# for running resiliency tests in a namespace.
+#
+# This template file is to be used in conjunction with the other template files
+#
+#   testrunner-k8s-sa.yml.template
+#   testrunner-k8s-rolebinding.yml.template
+#
+# within an automation pipeline and is not meant to be run separately or manually.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: testrunner-k8s-role
+  # A k8s namespace: e.g. terra-wsmtest, terra-ichang.
+  # Avoid using default or system namespaces such as kube-system.
+  namespace: terra-NAMESPACE
+  labels:
+    app.kubernetes.io/component: APP
+rules:
+  - apiGroups: [""]
+    resources: ["pods", "pods/exec"]
+    verbs: ["get", "list", "watch", "delete", "patch", "create", "update"]
+  - apiGroups: ["extensions", "apps"]
+    resources: ["deployments", "deployments/scale"]
+    verbs: ["get", "list", "watch", "delete", "patch", "create", "update"]
+```
+<summary>testrunner-k8s-rolebinding.yml.template</summary>
+
+```text
+# Do not modify this template file.
+
+# This template file is used for binding a Test Runner K8S Service Account
+# to a privileged RBAC role for running resiliency tests in a namespace.
+#
+# This template file is to be used in conjunction with the other template files
+#
+#   testrunner-k8s-sa.yml.template
+#   testrunner-k8s-role.yml.template
+#
+# within an automation pipeline and is not meant to be run separately or manually.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: testrunner-k8s-sa-rolebinding
+  # A k8s namespace: e.g. terra-wsmtest, terra-ichang.
+  # Avoid using default or system namespaces such as kube-system.
+  namespace: terra-NAMESPACE
+  labels:
+    app.kubernetes.io/component: APP
+subjects:
+  # Authorize In-Cluster Service Account
+  - kind: ServiceAccount
+    name: testrunner-k8s-sa
+    namespace: terra-NAMESPACE
+roleRef:
+  kind: Role
+  name: testrunner-k8s-role
+  apiGroup: rbac.authorization.k8s.io
+```
+
```

Comment:
It would be good to show actually running the tests and looking at the output, troubleshooting, etc.